Documentation/operations/performance/tuning.rst

```diff
@@ -144,6 +144,54 @@ To validate whether your installation is running with Bandwidth Manager,
 run ``cilium status`` in any of the Cilium pods and look for the line
 reporting the status for "BandwidthManager" which should state "EDT with BPF".
 
+BBR congestion control for Pods
+===============================
+
+The base infrastructure around MQ/FQ setup provided by Cilium's Bandwidth Manager
+also allows for use of TCP `BBR congestion control <https://queue.acm.org/detail.cfm?id=3022184>`_
+for Pods. BBR is in particular suitable when Pods are exposed behind Kubernetes
+Services which face external clients from the Internet. BBR achieves higher
+bandwidths and lower latencies for Internet traffic, for example, it has been
+shown that BBR's throughput can reach as much as 2,700x higher than today's best
+loss-based congestion control; queueing delays can be 25x lower.
```

Comment:
Any reference for the numbers?

[EDIT] Found it in the other document. I'd reuse the link here as well, because the numbers are high so it's easy to doubt them, especially when you're specifically looking for options to tune performance.

Reply:
Agree, good idea!